sdk/keyvault/keyvault-secrets/package.json

```diff
@@ -73,6 +73,7 @@
     "@azure/core-arm": "1.0.0-preview.3",
     "@azure/core-http": "1.0.0-preview.3",
     "@azure/core-paging": "1.0.0-preview.2",
+    "@azure/core-tracing": "1.0.0-preview.2",
```

Comment:
I believe this direct dependency is not needed as we are using exported entities from core-http